src/org/opendatakit/briefcase/operations/Export.java

```diff
@@ -22,157 +22,94 @@
   private static final Param<Void> OVERWRITE = Param.flag("oc", "overwrite_csv_export", "Overwrite files during export");
-  private static final Param<String> PEM_FILE = Param.arg("pf", "pem_file", "PEM file for form decryption");
-
-  public static Date toDate(String s) {
-    try {
-      return DATE_FORMAT.parse(s);
-    } catch (ParseException e) {
-      LOGGER.error("bad date range", e);
-      return null;
-    }
-  }
-
+  private static final Param<Path> PEM_FILE = Param.arg("pf", "pem_file", "PEM file for form decryption", Paths::get);
 
   public static Operation EXPORT_FORM = Operation.of(
       EXPORT,
       args -> export(
           args.get(STORAGE_DIR),
           args.get(FORM_ID),
-          args.get(FILE),
           args.get(EXPORT_DIR),
-          args.getOrNull(START),
-          args.getOrNull(END),
+          args.get(FILE),
           !args.has(EXCLUDE_MEDIA),
           args.has(OVERWRITE),
+          args.getOptional(START),
+          args.getOptional(END),
           args.getOptional(PEM_FILE)
       ),
       Arrays.asList(STORAGE_DIR, FORM_ID, FILE, EXPORT_DIR),
       Arrays.asList(PEM_FILE, EXCLUDE_MEDIA, OVERWRITE, START, END)
   );
 
-  public static void export(String storageDir, String formid, String fileName, String exportPath, Date startDateString, Date endDateString, boolean exportMedia, boolean overwrite, Optional<String> pemKeyFile) {
+  public static void export(String storageDir, String formid, Path exportPath, String baseFilename, boolean includeMediaFiles, boolean overwriteFiles, Optional<LocalDate> startDate, Optional<LocalDate> endDate, Optional<Path> maybePemFile) {
+    CliEventsCompanion.attach(log);
     bootCache(storageDir);
-    BriefcaseFormDefinition formDefinition = null;
-    List<BriefcaseFormDefinition> forms = FileSystemUtils.getBriefcaseFormList();
-    for (int i = 0; i < forms.size(); i++) {
-      BriefcaseFormDefinition x = forms.get(i);
-      if (formid.equals(x.getFormId())) {
-        formDefinition = x;
-        break;
-      }
-    }
+    Optional<BriefcaseFormDefinition> maybeFormDefinition = FileSystemUtils.getBriefcaseFormList().stream()
+        .filter(form -> form.getFormId().equals(formid))
+        .findFirst();
 
-    if (formDefinition == null) {
-      LOGGER.error("Form not found");
-      return;
-    }
+    BriefcaseFormDefinition formDefinition = maybeFormDefinition.orElseThrow(() -> new FormNotFoundException(formid));
 
     if (formDefinition.isFileEncryptedForm() || formDefinition.isFieldEncryptedForm()) {
-      File pemFile;
-      if (!pemKeyFile.isPresent()) {
-        LOGGER.error("Briefcase action failed: No specified PrivateKey file for encrypted form");
-        return;
-      }
-      pemFile = new File(pemKeyFile.get());
-      if (!pemFile.exists()) {
-        LOGGER.error("Briefcase action failed: No PrivateKey file for encrypted form");
-        return;
-      }
+      Path pemFile = maybePemFile
+          .filter(Files::notExists)
```

Comment:
Whoops! I think we got this backwards. You want to keep it if the file exists.
`.filter(Files::exists)`

Reply:
Fixed!